### DISCUSSION TITLE
023000ee ilegal after patch installation

### DISCUSSION THREAD

**Discussion by paulinhapirolas** [Mr-MIBonk/M.I.B._More-Incredible-Bash#157] [General]
Hi, after install the patch for MMI the 023000ee appears ilegal and I can't install the map update. I made swrl_clean and install again, but nothing happens. I formated the Nav_DB and right now I have the licence ilegal and no database maps.
What can I do? 
![IMG_20230324_074730](https://user-images.githubusercontent.com/128599265/227471175-4a159c1e-3d6d-4265-b26f-3bada7612ae0.jpg)
![IMG_20230324_074749](https://user-images.githubusercontent.com/128599265/227471182-f9b0ef89-6da3-47a9-8dfc-78fd61521629.jpg)
![IMG_20230324_075708](https://user-images.githubusercontent.com/128599265/227471186-0bc80c52-1fa2-45f1-a63a-f4bf4aabc2a4.jpg)

**Comment by harman-f** [comment]
Share log file

**Comment by paulinhapirolas** [comment]
[MU1438-JTB-07413.11.1511700A6E.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11061104/MU1438-JTB-07413.11.1511700A6E.log)

**Comment by paulinhapirolas** [comment]
I think that my MMI as another patch and it causes my problem
![IMG_20230324_100205](https://user-images.githubusercontent.com/128599265/227506461-e3f20e0b-19c4-4701-8e9f-715058dc2ff6.jpg)

**Reply by harman-f** [reply]
Yes, that is correct.

that checksum shown does not fit stock FW or M.I.B based patch.

I woul recommend to reinstall the FW full SWDL and patch afterwards with M.I.B

**Reply by paulinhapirolas** [reply]
And how can I re-install the FW k3346 if is the more higher. I tried with the red menu  but all appear to [N] instead [Y] and couldn't start the installation. How I proceed to a downgrade or another option. Thanks

**Comment by paulinhapirolas** [comment]
Hi, problem solved. The problem was that my MMI was already patched with another patch, so I renamed the ifs-rootpatch-00xba0000.9c85ee and everything works great
![IMG_20230325_171944](https://user-images.githubusercontent.com/128599265/228194487-a3bf8d00-2163-4d1d-bdd1-df3d9901750e.jpg)
![IMG_20230325_190012](https://user-images.githubusercontent.com/128599265/228194744-62aa2337-896f-488c-ae04-22f2af5b2147.jpg)

**Reply by flashroyal88** [reply]
have the same issue. could you help me out to know what name to give when you rename the ifc?

**Reply by paulinhapirolas** [reply]
In my case. I put this patch name. Because my MMI was already patched. Try to see you patch name in this part of the patch installer and rename
![image](https://user-images.githubusercontent.com/128599265/234073187-c1f5b9c0-0413-4911-87f8-4d8f0246c92a.png)

**Reply by flashroyal88** [reply]
> In my case. I put this patch name. Because my MMI was already patched. Try to see you patch name in this part of the patch installer and rename 


MU1326-ifs-root-part2-0x00ba0000-C005F400 this is my patch name and what do you think I should rename it to?

**Reply by paulinhapirolas** [reply]
Rename the patch to coincide . And then try to install. Yf goes OK the fec 023000ee should appear OK and not ilegal. In my case only the last part was diferent

**Reply by flashroyal88** [reply]
> Rename the patch to coincide . And then try to install. Yf goes OK the fec 023000ee should appear OK and not ilegal. In my case only the last part was diferent

Alright, thanks a lot. Will let you know if it worked :)

**Reply by flashroyal88** [reply]
Didn't work with me unfortunately.  Used name ifs-rootpatch-00xba0000.C005F4

**Reply by paulinhapirolas** [reply]
You rename the patch on the mib patch paste And then you try to install the patch to see if work

**Reply by flashroyal88** [reply]
Exactly. Still illegal

**Reply by paulinhapirolas** [reply]
And you try to upload the Fec manually?

**Reply by harman-f** [reply]
Share your log file

**Reply by paulinhapirolas** [reply]
It's on my 4th post

**Reply by flashroyal88** [reply]
I add Fec through M.I.B every time. Not sure if I know how to perform that manually

**Reply by flashroyal88** [reply]
> It's on my 4th post

That is how I did it

**Reply by paulinhapirolas** [reply]
I only know through the M. I. B

**Reply by flashroyal88** [reply]
> Share your log file

[MU1326-JTH-07605.02.1811850469.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11325586/MU1326-JTH-07605.02.1811850469.log)

**Comment by flashroyal88** [comment]
Turn out to be my own fault. I was going through 'add_deccontiner.fec' instead of 'flash patched image'. Key is legal now

**Reply by paulinhapirolas** [reply]
I'm glad for you. Another solution for whom try to solve similar problems.

**Comment by flashroyal88** [comment]
Hi there again. Another hurdle occurred. Whilst trying to install maps I receive warning. Any clues? Should I just ignore it and carry on?
 
![nav_update_wrning](https://user-images.githubusercontent.com/87669489/235318491-d0fd1a10-14ca-4c5d-bd24-bfbd5c45d5bc.jpg)

**Reply by paulinhapirolas** [reply]
Don't continue. I think you should download the maps from vw site and install by mmi update. That error I never see it before

**Reply by flashroyal88** [reply]
MMI says not compatible or older version. Will try downloading one from VW

**Reply by paulinhapirolas** [reply]
Try golf 7 from vw site. Its the map 2023 with the 28GB size

**Reply by harman-f** [reply]
As mentioned before, have not seen this issue before.
Maybe MIB2 TG channel can help

**Comment by flashroyal88** [comment]
Finally result. Latest release of maps installed flawlessly.

**Comment by paulinhapirolas** [comment]
Glad for tou

Enviado de Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
